### PR TITLE
Military family checkbox

### LINKED ISSUE #120: Add a field to the Families form that identifies a family as military veteran family.
# Summary
It is important to several of our diaper bank users and partners to know which of the families using their services are military or veteran families.

# Criteria for Completion
 - [ ] On the family form there should be field with a yes/no radio button with the label Military Family.

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
Resolves #120

### Description
Adds military family checkbox

### Type of change
* New feature (non-breaking change which adds functionality)

### Screenshots

Screen Shot 2019-06-29 at 3 12 34 PM
Screen Shot 2019-06-29 at 3 12 28 PM

### CODE REVIEW

**Comment by seanmarcia** [comment]
Hey Amina and Chase this is great but one small request. Could you change it from a check box to a Yes/No radio buttons? That makes it easier for our stakeholders to understand!

File: README.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ From the root of the app, run `bundle exec rails db:seed`. This will create some
 To login, use these default credentials provided in the seeds:
 
     Verified Organization
-      Email: org_admin1@example.com
+      Email: verified@example.com
```
Comment (benreyn) [inline README.md]: Thanks for this Documentation cleanup! 👍

File: app/views/families/index.html.erb [hunk 2]
```diff
@@ -12,8 +12,7 @@
     <% if notice %>
       <div class="alert alert-dismissible alert-success">
         <button class="close" type="button" data-dismiss="alert">×</button>
-        <p><%= notice %><a class="alert-link" href="#"></p>
-        </a></p>
+        <p><%= notice %><a class="alert-link" href="#"></a></p>
```
Comment (benreyn) [inline app/views/families/index.html.erb]: I assume this indentation fix was something your editor did automatically on save? It makes me wonder if we should add an [`.editorconfig`](https://editorconfig.org/) file to the repo (It also makes me wonder if editorconfig would catch this sort of thing)

Nonetheless, thanks for the cleanup!

Comment (chaserx) [reply app/views/families/index.html.erb]: removed the extra closing `p` tag there.

Comment (benreyn) [reply app/views/families/index.html.erb]: Ahh I see it now. Was hard to see what was going on.

**Comment by chaserx** [comment]
Updated the family form with  a Yes / No radio button

**Comment by benreyn** [comment]
Thanks @chaserx! Ill wait for CI to pass and then merge 👍

**Pull request merged**